src/js/model/validateCarName.js

```diff
@@ -0,0 +1,8 @@
+import { CAR_NAME_MAX_LENGTH, CAR_NAME_SEPARATOR, errorMessages } from '../constants.js';
+
+export default function validateCarName(name) {
+  const names = name.split(CAR_NAME_SEPARATOR);
+  const isLessThanFiveCharacters = (characters) => characters.length <= CAR_NAME_MAX_LENGTH;
+
+  if(!names.every(isLessThanFiveCharacters) || !name) throw new Error(errorMessages.INVALID_CAR_NAMES);
```

Comment:
```suggestion
  if(!name || !names.every(isLessThanFiveCharacters)) throw new Error(errorMessages.INVALID_CAR_NAMES);
```
name을 토대로 names를 만들고 있기 때문에, names보다 name을 먼저 검증해야 하지 않을까요?